app/models/user.rb

```diff
@@ -201,4 +209,16 @@ def self.configure_buried_profile!(profile)
   def self.buried_profile
     (@buried_profile || {}).slice(:first_name, :last_name, :email)
   end
+
+  def placeholder_image_url
+    PLACEHOLDER_IMAGE_URL
+  end
+
+  def current_organic_context
```

Comment:
I want to hear thoughts about this one too.
I think this can be generalized, specially with the introduction of `current_workspace`.
Although the `course` is not involved here, it might be.